src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -81,45 +109,176 @@ function dismissWorkspaceError(policyID: string, pendingAction: OnyxCommon.Pendi
+                        style={[styles.mh5, styles.mb3]}
+                        disabled={item.disabled}
+                        onPress={item.action}
+                    >
+                        {({hovered}) => (
+                            <WorkspacesListRow
+                                title={item.title}
+                                menuItems={threeDotsMenuItems}
+                                workspaceIcon={item.icon}
+                                ownerAccountID={item.ownerAccountID}
+                                workspaceType={item.type}
+                                rowStyles={hovered && styles.hoveredComponentBG}
+                                layoutWidth={isSmallScreenWidth ? CONST.LAYOUT_WIDTH.NARROW : CONST.LAYOUT_WIDTH.WIDE}
+                                brickRoadIndicator={item.brickRoadIndicator}
+                            />
+                        )}
+                    </PressableWithoutFeedback>
+                </OfflineWithFeedback>
+            );
+        },
+        [isSmallScreenWidth, styles.mb3, styles.mh5, styles.ph5, styles.hoveredComponentBG, translate],
     );
 
+    const listHeaderComponent = useCallback(() => {
+        if (isSmallScreenWidth) {
+            return <View style={styles.mt5} />;
+        }
+
+        return (
+            <View style={[styles.flexRow, styles.gap5, styles.mh5, styles.mv5, styles.pl5]}>
+                <View style={[styles.flexRow, styles.flex1]}>
+                    <Text
+                        numberOfLines={1}
+                        style={[styles.flexGrow1, styles.textLabelSupporting]}
+                    >
+                        {translate('workspace.common.workspaceName')}
+                    </Text>
+                </View>
+                <View style={[styles.flexRow, styles.flex1, styles.workspaceOwnerSectionTitle]}>
+                    <Text
+                        numberOfLines={1}
+                        style={[styles.flexGrow1, styles.textLabelSupporting]}
+                    >
+                        {translate('workspace.common.workspaceOwner')}
+                    </Text>
+                </View>
+                <View style={[styles.flexRow, styles.flex1, styles.workspaceTypeSectionTitle]}>
+                    <Text
+                        numberOfLines={1}
+                        style={[styles.flexGrow1, styles.textLabelSupporting]}
+                    >
+                        {translate('workspace.common.workspaceType')}
+                    </Text>
+                </View>
+                <View style={[styles.ml10, styles.mr2]} />
+            </View>
+        );
+    }, [isSmallScreenWidth, styles, translate]);
+
+    const policyRooms = useMemo(() => {
+        if (!reports || isEmptyObject(reports)) {
+            return;
+        }
+
+        return Object.values(reports).reduce<ChatPolicyType>((result, report) => {
+            if (!report?.reportID || !report.policyID) {
```

Comment:
Hey folks👋
This has caused a regression in https://github.com/Expensify/App/issues/51820, user is navigated to thread message when going to `#announce/admins` room
Since all reports in `announce` room have `chatType === policyAnnounce` (even the thread reports), we were mistakenly identifying any report with `chatType === policyAnnounce` as the announce room in the switch statement below
This was resolved by returning early here is a report has `parentReportID`